polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Address Formats
+description: Overview of Polkadot address formats, covering different encoding strategies and best practices for secure and efficient usage.
+---
+
+# Address Formats
+
+The default Polkadot SDK address format is `SS58`, a modified version of Bitcoin's `Base58Check` format tailored for Polkadot SDK-based chains. 
+
+While other formats can be used, `SS58` provides a base-58 encoded value that uniquely identifies an account on any Polkadot SDK chain. 
+
+It's designed to be extensible, accommodating different ways of identifying accounts across various chains.
+
+## Basic Format
+
+You can find the implementation for the `SS58` address format in [Ss58Codec](https://paritytech.github.io/polkadot-sdk/master/sp_core/crypto/trait.Ss58Codec.html){target=\_blank}.
+
+The basic format of the address can be described as:
+
+```
+base58encode ( concat ( <address-type>, <address>, <checksum> ) )
+```
+
+The address is the concatenated byte series consisting of an address type, the encoded address, and a checksum that is passed into a `Base58` encoder. 
+
+The `Base58encode` function is implemented exactly as defined in Bitcoin and IPFS specifications, using the same alphabet as both of those implementations. 
+
+The `Base58` alphabet eliminates characters that might look ambiguous when printed, for example:
+
+- Non-alphanumerical characters (+ and /)
+- Zero (0)
+- Capital i (I)
+- Capital o (O)
+- Lower-case L (l)
+
+## Address Type
+
+The address type in the `SS58` address format consists of one or more bytes that specify the exact format of the address bytes that follows it.
+
+Currently, the valid values are:
+
+- `00000000b..=00111111b` (0 to 63 inclusive)
+
+    Simple account/address/network identifier. The byte can be interpreted directly as such an identifier.
+
+- `01000000b..=01111111b` (64 to 127 inclusive)
+
+    Full address/address/network identifier. The lower 6 bits of this byte represent the upper 6 bits of a 14-bit identifier, while the next byte defines the lower 8 bits. This allows for identifiers ranging from 0 to 16,383 (2²¹).
+
+- `10000000b..=11111111b` (128 to 255 inclusive)
+
+    This section is reserved for future address format extensions. The address type 42 is designed to be valid across all Polkadot SDK networks that support fixed-length addresses. 
```

Comment:
Seems like some weird spacing / tab here, not sure, maybe best to remove it (unless it renders fine, then ignore)

Reply:
I'll refer @dawnkelly09 @eshaben to this comment. Not sure what to do here